deployment/stacks/ui.ts

```diff
@@ -172,7 +178,6 @@ function setupWaf(
     "CloudfrontWafConstruct",
     {
       name: `${project}-${stage}-ui`,
-      blockByDefault: false,
```

Comment:
Surprised by this being in here.

Reply:
Oh yeah I forgot to ask about this. I removed it because it wasn't in `WafProps`, so I wasn't sure what should be done.